src/quart_db/backends/asyncpg.py

```diff
@@ -159,7 +159,8 @@ async def connect(self) -> None:
             self._pool = await asyncpg.create_pool(dsn=self._url, init=self._init)
 
     async def disconnect(self, timeout: Optional[int] = None) -> None:
-        await asyncio.wait_for(self._pool.close(), timeout)
+        if self._pool:
```

Comment:
Could you change this to `if self._pool is not None`?

Reply:
Do'h! Of course!